src/video_core/macro/macro.cpp

```diff
@@ -15,6 +19,22 @@
 
 namespace Tegra {
 
+static void Dump(u64 hash, std::span<const u32> code) {
+    const auto base_dir{Common::FS::GetYuzuPath(Common::FS::YuzuPath::DumpDir)};
+    const auto macro_dir{base_dir / "macros"};
+    if (!Common::FS::CreateDir(base_dir) || !Common::FS::CreateDir(macro_dir)) {
+        LOG_ERROR(Common_Filesystem, "Failed to create macro dump directories");
+        return;
+    }
+    const auto name{macro_dir / fmt::format("{:016x}.macro", hash)};
+    std::fstream macro_file(name, std::ios::out | std::ios::binary);
+    if (!macro_file) {
+        LOG_ERROR(Common_Filesystem, "Unable to open or create file at {}", name.c_str());
```

Comment:
Because MSVC STL chokes on UTF8 conversion

```suggestion
        LOG_ERROR(Common_Filesystem, "Unable to open or create file at {}", Common::FS::PathToUTF8String(name));
```